### PR TITLE
Added 'Remember me' option to the login form #971

### LINKED ISSUE #971: (SP:) [Login popup] Verify that 'Remember me' function and icon are present
**Priority:** medium

**Description:** this test verifies that the user can apply 'Remember me' function on the Login popup

**Preconditions** user is already registered on https://s2s-front-stage.azurewebsites.net/ website

**Steps to reproduce**
1. Click Login button in the upper right corner of the header 
image

2. Verify that 'Remember me' icon is present at the Login popup ( 'Remember me' icon should locate under 'Password' field on the left side, the opposite to 'Forgot password' link) 
image

**Expected result**
'Remember me' icon is presented in the Login popup according to the mockup 
image

**Actual result**
There is no 'Remember me' icon 

Epic: #32

### PR DESCRIPTION
**Pull request by Olenka-Hryk**
#### Added 'Remember me' option to the login form [ Close #971 ]

Now the '`Remember me`' checkbox is included in the login popup, following the mockup in [Figma](https://www.figma.com/design/liJZDYhUnDP15Pi9bipDcv/Space2Study-(Students)?node-id=2-24753&t=yFp3udW5yyzK561A-4)

| Before changing | After changing |
|-----------------|----------------|
| Before | After |

### CODE REVIEW

**Review by docia — APPROVED** [review]

**Review by Mygal-Alex — APPROVED** [review]

**Review by YaroslavChuiko — APPROVED** [review]

File: src/containers/guest-home-page/login-form/LoginForm.jsx [hunk 1]
```diff
@@ -5,6 +5,8 @@ import { useSelector } from 'react-redux'
 import Box from '@mui/material/Box'
 import ButtonBase from '@mui/material/ButtonBase'
 import Typography from '@mui/material/Typography'
+import Checkbox from '@mui/material/Checkbox'
+import FormControlLabel from '@mui/material/FormControlLabel'
```
Comment (Mav-Ivan) [inline src/containers/guest-home-page/login-form/LoginForm.jsx:9]: implement enter after libs

File: src/containers/guest-home-page/login-form/LoginForm.styles.js [hunk 2]
```diff
@@ -4,13 +4,27 @@ export const styles = {
     flexDirection: 'column',
     minWidth: { sm: '340px' }
   },
+  loginOptionsContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    height: '40px',
+    padding: '0px 5px',
```
Comment (Mav-Ivan) [inline src/containers/guest-home-page/login-form/LoginForm.styles.js:12]: better to write padding as `p:'0px 5px',`

File: src/containers/guest-home-page/login-form/LoginForm.jsx (hunk 1 above)
Comment (Olenka-Hryk) [reply src/containers/guest-home-page/login-form/LoginForm.jsx:9]: Updated as requested

File: src/containers/guest-home-page/login-form/LoginForm.styles.js (hunk 2 above)
Comment (Olenka-Hryk) [reply src/containers/guest-home-page/login-form/LoginForm.styles.js:12]: Thanks. I have made the corrections

**Pull request merged**